src/ontology/mondo-edit.obo

```diff
@@ -423292,6 +423291,7 @@ relationship: excluded_subClassOf MONDO:0019189 {source="Orphanet:468726", sourc
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/3448" xsd:anyURI
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/4521" xsd:anyURI
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/4985" xsd:anyURI
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8062" xsd:string
```

Comment:
This is very weird... It is a "term" but it has the x-ref... I will take a look in the obo file